UM-USElab/gradecraft-development
# Marks badges that are locked as unavailable to be awarded

### Status
**READY (post build)**

### Description
Instructors have asked that we make awarding badges that are locked as unavailable in the grading form. This closes #3752 

### Migrations
NO

### Steps to Test or Reproduce
Navigate to a grading form in GC 101 - Course with badges. Confirm that the badge Unlocked by Badge is greyed out and unavailable to award (no checkbox). 


### Impacted Areas in Application
List general components of the application that this PR will affect:

* Grading form 

======================
Closes #3752
